client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/hooks.tsx

```diff
@@ -108,3 +109,57 @@ export function useResendVerificationCode(): {
 		},
 	} );
 }
+
+export function useContactModalTitleAndSubtitle(
+	type: 'email' | 'phone',
+	action: AllowedMonitorContactActions
+): {
+	title: string;
+	subtitle: string;
+} {
+	const translate = useTranslate();
+
+	const getContactModalTitleAndSubTitle = useMemo(
+		() => ( {
+			email: {
+				add: {
+					title: translate( 'Add new email address' ),
+					subtitle: translate( 'Please use only your number or one you have access to.' ),
+				},
+				edit: {
+					title: translate( 'Edit your email address' ),
+					subtitle: translate( 'If you update your email address, you’ll need to verify it.' ),
+				},
+				remove: {
+					title: translate( 'Remove Email' ),
+					subtitle: translate( 'Are you sure you want to remove this email address?' ),
+				},
+				verify: {
+					title: translate( 'Verify your email address' ),
+					subtitle: translate( 'We’ll send a code to verify your email address.' ),
+				},
+			},
+			phone: {
+				add: {
+					title: translate( 'Add your phone number' ),
+					subtitle: translate( 'Please use an accessible phone number. Only alerts sent.' ),
```

Comment:
This also feels a little off, mostly because of the use of the word accessible.  I feel like the phrasing in the email subtitle would work better here (your own number or one you have access to)